src/bootstrap/format.rs

```diff
@@ -20,7 +20,12 @@ fn rustfmt(src: &Path, rustfmt: &Path, path: &Path, check: bool) {
     cmd.arg(&path);
     let cmd_debug = format!("{:?}", cmd);
     let status = cmd.status().expect("executing rustfmt");
-    assert!(status.success(), "running {} successful", cmd_debug);
+    assert!(
+        status.success(),
+        "Running `{}` failed. \
+        If you're running `tidy`, try again with `--bless` flag.",
+        cmd_debug,
+    );
```

Comment:
While we're at it, could we change this to an if + `std::process::exit(1)`? Bit nicer than the assert! message probably.

We should also explicitly mention `x.py fmt` probably...

Reply:
> While we're at it, could we change this to an if + `std::process::exit(1)`? Bit nicer than the assert! message probably.

Indeed, I'll do it.

> We should also explicitly mention `x.py fmt` probably...

As follows?

```
If you just want to format code, run `x.py fmt` instead.
```

Reply:
Sure!

Reply:
Updated.